### PR TITLE
Add Ole Petter's banned e-mail and remove Kristian from reviewers

### PR DESCRIPTION
**Pull request by oleorhagen**
Untested ofc

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -34,10 +35,13 @@
       "username": "mender-bottom",
       "secrets": "mender-secrets.json",
       "trusted": ["oleorhagen", "kacf", "pasinskim",  "olehermanse"],
-      "reviewers": ["oleorhagen", "kacf"],
+      "reviewers": ["oleorhagen"],
```
Comment (olehermanse) [inline config.json]: **Note:**

1. The pinging for reviewers feature, which this list is used for, is now disabled by default. You have to add `"ping_reviewer"` to `"bot_features"` to enable it.
2. If it was enabled, this is just the fallback / default list. There is a list for the mender client repo below, where you can also remove kristian, if you want.

File: config.json [hunk 2]
```diff
@@ -34,10 +35,13 @@
...
       "jenkins": "https://mender-jenkins.mender.io/",
       "jenkins_job": "mender-builder",
       "bot_features": ["create_pr_magic"],
+        "banned_emails": {
+            "ole1": "2c04264ee57ffe7708855bccdf11d7c0e5f797c7556cf0b7b710fffdc5e8a925"
+        }
```
Comment (olehermanse) [inline config.json]: Wrong indentation and comma missing. Get a better editor ;)

File: config.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
           "craig": "5a09cedd03ac8f6625afbf82320d9182ab56e507311f43d02aace8f5fe6d93f5",
           "igor": "56d20f3d01cf0c08db466fb629187ff41a30bb170623251b7e14e32c3c1a67a4",
           "vratislav": "54a458f06da35064562b258e52f5b233dbbd561a51816097b7bce1a306775d05",
-          "ole": "94534ae37a66ade3bae446c81cc15433efcda3a8516ffe6b2c2e0f6dd70127dd"
+          "ole": "94534ae37a66ade3bae446c81cc15433efcda3a8516ffe6b2c2e0f6dd70127dd",
+          "ole1": "2c04264ee57ffe7708855bccdf11d7c0e5f797c7556cf0b7b710fffdc5e8a925"
```
Comment (olehermanse) [inline config.json]: Did you create the hash using:

```
$ echo -n "my-email@gmail.com" | shasum -a 256
```

**Note** the `-n` option to echo to leave off the newline.

( I don't know your email, so I don't know if you did it correctly, just figured I'd check ).

File: config.json [hunk 4]
```diff
@@ -38,6 +39,10 @@
       "jenkins": "https://mender-jenkins.mender.io/",
       "jenkins_job": "mender-builder",
       "bot_features": ["create_pr_magic"],
+      "banned_emails": {
+          "ole1": "23d337c7f69fee4a04dca86de41cf737a0c585029864eab34aa1a5390aa7b845"
+
```
Comment (vpodzime) [inline config.json]: Extra blank line.

Comment (oleorhagen) [reply config.json]: Lol, *blush*

File: config.json [hunk 5]
```diff
@@ -34,13 +35,16 @@
       "username": "mender-bottom",
       "secrets": "mender-secrets.json",
       "trusted": ["oleorhagen", "kacf", "pasinskim",  "olehermanse"],
-      "reviewers": ["oleorhagen", "kacf"],
+      "reviewers": ["oleorhagen"],
```
Comment (olehermanse) [inline config.json]: @oleorhagen Tom won't know what to do when you open a PR.

**Pull request merged**